stumpy/core.py

```diff
@@ -2559,3 +2559,76 @@ def _select_P_ABBA_value(P_ABBA, k, custom_func=None):
             MPdist = partition[k]
 
     return MPdist
+
+
+@njit(parallel=True)
+def _merge_topk_PI(PA, PB, IA, IB):
+    """
+    Merge two top-k matrix profiles PA and PB, and update PA (in place) while
+    always choosing values of PA over values of PB in case of ties. Also, update
+    IA accordingly.
+
+    Parameters
+    ----------
+    PA : numpy.ndarray
+        a (top-k) matrix profile, with ndim of 2, where values in each row are
```

Comment:
Please make the first letter uppercase "A (top-k) matrix profile".

Repeat/review everywhere

Reply:
Will do!